### PR TITLE
feat: factor in get businesses endpoint

### PR DESCRIPTION
**Pull request by idorenyinudoh**

### CODE REVIEW

File: lib/Thepeer.js [hunk 1]
```diff
@@ -157,6 +157,25 @@ class ThePeer {
...
     }
   }
+
+  /**
+   * Get a list of all integrated businesses and their respective images
+   * @param {string} channel - Channel type (send, checkout, direct_charge)
+   * @returns {JSON} A JSON response containing the details of the businesses
+   * @memberof Thepeer
+   */
+  async getBusinesses (channel) {
```
Comment (kamsy) [inline lib/Thepeer.js:167]: maintain the same pattern at the other functions

**Review by kamsy — APPROVED** [review]
LGTM!

**Comment by ichtrojan** [comment]
lmao

**Comment by kamsy** [comment]
> lmao

You already know 😂

**Pull request merged**